### DISCUSSION TITLE
Modify Comparison of tag argument

### DISCUSSION THREAD

**Discussion by DcNathanMaroko** [rfennell/AzurePipelines#1495] [Q&A]
Hi, I was just wondering if there was a way to override the comparison logic when using the tag argument. For instance, instead of matching the exact tag with the name 'release', I'd like to match 'release' to a tag named 'releasev1.0.1' or something similar. Is there a way to do that?

**Comment by rfennell** [comment]
There is no override logic for tag matching.

You would have to modify the existing logic in ReleaseNotesFunctions.ts lne 1200
 
```js
if (tags.length === 0 ||
   (tags.length > 0 && build.tags.sort().join(",") === tags.sort().join(",")) ||
   (tags.find(t => t === "*") && tags.length > 0 && tags.filter((t) => t === "*" || build.tags.find((bt) => bt === t)).length === tags.length)) {
     agentApi.logInfo("Considering build");
```
